Farmacy/Delegate/AppDelegate.m

```diff
@@ -31,7 +31,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
         
         configuration.applicationId = [dict objectForKey:@"parse_app_id"];
         configuration.clientKey = [dict objectForKey:@"parse_client_key"];
-        configuration.server = @"https://parseapi.back4app.com";
+        configuration.server = @"wss://farmacy.b4a.io";
```

Comment:
This was done in another PR right? You will need to resolve the conflicts before merging